proto/bpfman.proto

```diff
@@ -110,7 +110,7 @@ message KprobeAttachInfo {
     string fn_name = 1;
     uint64 offset = 2;
     bool retprobe = 3;
-    optional string namespace = 4;
+    optional int32 namespace_pid = 4;
```

Comment:
Good point.  I don't mind changing this, but it may not be necessary yet.  First, the namespace option isn't supported yet.  There's no way to use it from kubernetes, and if someone tries it from bpfctl or bpfman, they'll get an error.  Second, we're going to be making major changes in 0.4.0, so I don't know if we need to worry about backward compatibiilty yet. Let's discuss during the meeting tomorrow.